galette/lib/Galette/Entity/Adherent.php

```diff
@@ -1268,7 +1268,7 @@ public function validate(string $field, $value, array $values): void
                     $d = \DateTime::createFromFormat(__("Y-m-d"), $value);
                     if ($d === false) {
                         //try with non localized date
-                        $d = \DateTime::createFromFormat("Y-m-d", $value);
+                        $d = \DateTime::createFromFormat("d/m/Y", $value);
```

Comment:
Well, in facts, `d/m/Y` should be the one tried for `__("Y-m-d")` (the localized version).
Maybe there is a gap; but this line should only act as a fallback to default database known format; when the localized version is not known/incorrect/whatever.

I cannot accept the patch, I'll take a look on my side trying to understand what goes wrong.

Reply:
As far as I understand; the problem is the date picker is not properly loaded. Could you precise which Galette version you use?

Reply:
hey @trasher, thanks for quick feedback on this.

I'm currently playing around with the latest nightly build (so of course: no support here ;))

The date picker is loading flawlessly as far as I can tell.
<img width="1678" alt="2023-02-28-15 09 41" src="https://user-images.githubusercontent.com/97628193/221878791-3e1d3293-5999-4158-b828-ca37e555c8f1.png">

It's rather the format the date picker generates the dates. They'll always match `dd/mm/YYYY` regardless of the current locale. I suspect this being the culprit here.
<img width="1678" alt="2023-02-28-15 09 55" src="https://user-images.githubusercontent.com/97628193/221879232-58b4de41-6ea8-48b1-ba15-6403790bef16.png">

Reply:
Yes, the date picker is in cause. 

![image](https://user-images.githubusercontent.com/224733/221887710-16b6a1a2-a1f7-4291-aa3c-492f666584be.png)

We can see here the format we ask for (`yyyy-mm-dd`) is not hte one in the picker (`dd/mm/yyyy`)

Reply:
Hi! 

The culprit is the date formatter of the calendar component. For the moment, it is not yet translated (nor really adapted to the required input format I think) : https://github.com/galette/galette/blob/ba185008162bb721e3459b18ac78bb2e17119403/galette/templates/default/elements/js/calendar.js.twig#L61

Since FUI 2.9, it is now possible to use custom formats : 
https://fomantic-ui.com/modules/calendar.html#custom-format
https://fomantic-ui.com/modules/calendar.html#/settings

Reply:
I found the issue. In `templates/default/elements/js/calendar.js.twig`, rthe `formatter` returns `return day + '/' + month + '/' + year;`. 
I do not know yet how to fix that